mucommander-core/build.gradle

```diff
@@ -24,8 +24,27 @@ dependencies {
     implementation 'ch.qos.logback:logback-classic:1.2.3'
     implementation 'org.jmdns:jmdns:3.5.5'
     implementation 'commons-collections:commons-collections:3.2.2'
+
     comprise 'org.unix4j:unix4j-command:0.6'
 
+    // The following libs are required by JediTerm.
+    // Since, I haven't found any repo where they are published,
+    // so they have been built from sources (https://github.com/JetBrains/jediterm, c21a2b27629dd3760c603edeb258f9a862e75d34)
+    // and placed directly into muC.
+    comprise files('libs/JediTerm-2.69.jar')
+    comprise files('libs/jediterm-typeahead-2.69-SNAPSHOT.jar')
+    comprise files('libs/kotlin-stdlib-common-1.5.10.jar')
+    comprise files('libs/annotations-20.1.0.jar')
+    comprise files('libs/jna-5.10.0.jar')
+    comprise files('libs/pty4j-0.12.7.jar')
+    comprise files('libs/terminal-2.69.jar')
+    comprise files('libs/eawtstub.jar')
+    comprise files('libs/jna-platform-5.10.0.jar')
+    comprise files('libs/purejavacomm-0.0.11.1.jar')
+    comprise files('libs/trove4j.jar')
+    comprise files('libs/jediterm-pty-2.69-SNAPSHOT.jar')
+    comprise files('libs/kotlin-stdlib-1.5.10.jar')
```

Comment:
> at least some of those resources should appear on maven central and specifically jna is [already used](https://github.com/mucommander/mucommander/blob/1.0.1/mucommander-commons-file/build.gradle#L8-L9) so it would be better to get it once and have it shared between the modules that need it
> 
> as for the ones that are specific to the terminal, are they all built from https://github.com/JetBrains/jediterm or where did you get them from?

I took them from jediterm build folder. I will try to use maven for those libs which are there.